common/persistence/cassandra/errors.go

```diff
@@ -41,7 +41,8 @@ var (
 	errorPriority = map[reflect.Type]int{
 		reflect.TypeOf(&p.ShardOwnershipLostError{}):             0,
 		reflect.TypeOf(&p.CurrentWorkflowConditionFailedError{}): 1,
-		reflect.TypeOf(&p.ConditionFailedError{}):                2,
+		reflect.TypeOf(&p.WorkflowConditionFailedError{}):        2,
+		reflect.TypeOf(&p.ConditionFailedError{}):                3,
```

Comment:
Is this map used for durable state anywhere? If so we probably want to keep ConditionFailedError as 2

Reply:
this is in mem only map, used to sort & return the most important error if there are > 1 errors